src/main/java/picocli/CommandLine.java

```diff
@@ -8370,7 +8383,12 @@ private static String inferLabel(String label, String fieldName, ITypeInfo typeI
                 public T converters(ITypeConverter<?>... cs) { this.converters = Assert.notNull(cs, "type converters").clone(); return self(); }
 
                 /** Sets a regular expression to split option parameter values or {@code ""} if the value should not be split, and returns this builder. */
-                public T splitRegex(String splitRegex)       { this.splitRegex = Assert.notNull(splitRegex, "splitRegex"); return self(); }
```

Comment:
Let's leave the existing `splitRegex(String)` setter as it is, and add a separate new setter `ArgSpec.Builder::usageSplitRegex(String)`.

We also need new getters `String ArgSpec.Builder::usageSplitRegex()` and  `String ArgSpec::usageSplitRegex()`.